util/util.go

```diff
@@ -174,6 +176,16 @@ func GetLoggingWriter(cfg *config.Config) (io.Writer, error) {
 			MaxAge:     28,   // days
 			Compress:   true, // disabled by default
 		}
+		// rotate log file on SIGHUP
+		ch := make(chan os.Signal, 1)
+		signal.Notify(ch, syscall.SIGHUP)
+		go func() {
+			for range ch {
+				if err := writer.(*lumberjack.Logger).Rotate(); err != nil {
+					fmt.Printf("failed to rotate log file: %v\n", err)
```

Comment:
You can use `log.Printf()` Even if we're rotating, the log entry will end up somewhere, not on stdout where it may just be lost.

Reply:
Yep, after I move the piece of code to `cmd/garm/main.go`, I'll have do that.

Reply:
Yep, after I move the piece of code to `cmd/garm/main.go`, I'll do that.